cfgov/v1/migrations/0006_update_tables.py

```diff
@@ -26,7 +26,7 @@ def update_content_tables(wagtail_page):
         if item['type'] == 'table':
             item['type'] = 'table_block'
             item['value'] = create_tableblock_data(old_table=item['value'])
-            print "Creating a new TableBlock on page %s" %(wagtail_page.title)
+            print "Creating a new TableBlock on page %s of page type %s" %(wagtail_page.title, wagtail_page.__class__.__name__)
```

Comment:
@vickumar1981 Mostly b/c we're not using a logger anywhere else, but I agree it's better to log than to print so I'll update it! Thanks.